core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1303,4 +1303,9 @@ package object config {
     .doc("Staging directory used while submitting applications.")
     .stringConf
     .createOptional
+
+  private[spark] val BUFFER_PAGESIZE = ConfigBuilder("spark.buffer.pageSize")
+    .bytesConf(ByteUnit.BYTE)
```

Comment:
can you add a doc?

Reply:
@HyukjinKwon 
Thanks for your time.
Now I added the doc and corrected the scalastyle issue

Reply:
I was going to say this needs a default, but it looks like the default in non-test code is not a single value, and different tests want different defaults.